code/modules/vending/subtype.dm

```diff
@@ -0,0 +1,90 @@
+/obj/machinery/vending/tool
```

Comment:
delete the copypasta. That's not the subtype vendor.

Reply:
bros im retarded